include/astshim/PolyMap.h

```diff
@@ -71,23 +74,38 @@ friend class Object;
 
             Each final output coordinate value is the sum of the terms described
             by the `ncoeff_f` columns in the supplied array.
+
+            If coeff_f is empty then no forward transformation is provided.
```

Comment:
These paragraph breaks are a bit problematic: https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/classast_1_1_poly_map.html#aa45fef22f3c82b8bb8e2549c5de097cd.

I'd recommend moving the discussion of how `coeff_*` needs to be formatted to the main description, and restricting the descriptions of `coeff_f` and `coeff_i` to something that will fit in a single paragraph.